autoload/smartinput.vim

```diff
@@ -71,32 +71,39 @@ function! smartinput#define_default_rules()  "{{{2
     call add(self.names, a:name)
     let self.table[a:name] = a:urules
   endfunction
+  if exists('g:smartinput_no_break_undo') || v:version < 800
```

Comment:
I believe nobody wants to explicitly disable this.  Let's remove this `exists()` check.

Reply:
No. I often see users who want to do undo in small units.

But sorry, this variable name was wrong.
I'm thinking the default is **no break undo**, If a variable exists (optional) to **break undo**.

How about changing the variable name to `g:smartinput_break_undo`?

Reply:
Changed at 6b34708.